projects/canopy/src/lib/forms/checkbox-group/checkbox-group.stories.ts

```diff
@@ -0,0 +1,85 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { FormBuilder, FormGroup, ReactiveFormsModule } from '@angular/forms';
+
+import { action } from '@storybook/addon-actions';
+import { boolean, text, withKnobs } from '@storybook/addon-knobs';
+import { moduleMetadata } from '@storybook/angular';
+
+import { CanopyModule } from '../../canopy.module';
+import { notes } from './checkbox-group.notes';
+
+@Component({
+  selector: 'lg-reactive-form',
+  template: `
+    <form [formGroup]="form">
+      <lg-checkbox-group [inline]="inline" formControlName="colors">
+        {{ label }}
+        <lg-hint *ngIf="hint">{{ hint }}</lg-hint>
+        <lg-toggle value="red">Red</lg-toggle>
```

Comment:
I guess in the next PR we will be adding multiple selectors to the LgToggleComponent? It will be interesting to see if we can avoid a breaking change without adding code smell.

Reply:
I think the only way we can avoid a breaking change is if we keep the current implementation with the variant being a `@Input` property, but I think the multiple selectors approach is cleaner and more intuitive so I would go with that. :)
I am also ok with the `@Input` implementation for the variant, if we want to avoid breaking changes.

Reply:
unless the `@Input` one can work as an override to the automatic one 🤔  let's see a breaking change isn't the end of the world.